test/java/seedu/addressbook/util/TypicalPersons.java

```diff
@@ -25,9 +27,14 @@ public TypicalPersons() {
             bill = new Person(new Name("Bill Clint"), new Phone("92229222", false), new Email("bc@gmail.com", false),
                     new Address("2 Clementi Road", true), Collections.emptySet());
             candy = new Person(new Name("Candy Destiny"), new Phone("93339333", true),
-                    new Email("cd@gmail.com", false), new Address("3 Clementi Road", true), Collections.emptySet());
+                    new Email("cd@gmail.com", false), new Address("3 Clementi Road", true), Collections.singleton(new Tag("test")));
+
+            // Preparation of Set of Tags for dan
+            Set<Tag> danTags = new HashSet<>();
```

Comment:
Moving forward, you can think about why `Collections.singleton()` was used to pass in the tag as an argument previously.